src/components/Dashboard/PendingTxsList/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { ReactElement } from 'react'
+import { List } from '@material-ui/core'
+import styled from 'styled-components'
+import { useSelector } from 'react-redux'
+
+import { Transaction } from 'src/logic/safe/store/models/types/gateway.d'
+import { sm } from 'src/theme/variables'
+import { currentChainId } from 'src/logic/config/store/selectors'
+import { generateSafeRoute, SAFE_ROUTES } from 'src/routes/routes'
+import { getChainById } from 'src/config'
+import PendingTxListItem from 'src/components/Dashboard/PendingTxListItem'
+import { currentSafe } from 'src/logic/safe/store/selectors'
+import { MAX_TXS_DISPLAY } from 'src/routes/Home'
+import Skeleton from '@material-ui/lab/Skeleton/Skeleton'
+
+const SkeletonWrapper = styled.div`
+  margin: ${sm} auto;
+  padding: 4px 16px;
+`
+
+const PendingTxsList = ({ transactions }: { transactions?: Transaction[] }): ReactElement => {
+  const { address } = useSelector(currentSafe)
+  const chainId = useSelector(currentChainId)
+
+  if (!transactions) {
+    return (
+      <List component="div">
+        {Array.from(Array(MAX_TXS_DISPLAY).keys()).map((key) => (
```

Comment:
I would not use basic numbers as keys. It is identical to using indexes.

Reply:
I'm just trying to render `MAX_TXS_DISPLAY` (5) Skeleton components here. It is very static so I don't see a problem using the basic numbers or even the indexes in this specific situation...
Do you have any alternative implementation suggestion?

Reply:
I'd normally suggest some form of joining the keys but as it's so simple then let's leave it. It's an anti-pattern nonetheless.